pipelines/official_build.yml

```diff
@@ -0,0 +1,32 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+stages:
+- stage: One
```

Comment:
what is stage One?

Reply:
I worried that use different stages maybe run on different node. 
This worry comes from log, and haven't confirmed by ms doc.
So I prefer to use one stage. it is a non-sense name

Reply:
changed to Build